index.html

```diff
@@ -0,0 +1,46 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-bar">
+            <div class="page--logo">
+                <a class="page--link" href="#">My Logo</a>
+            </div>
+            <div class="nav--menu">
+                    <div></div>
+                    <div></div>
+                    <div></div>
+            </div>
+        </nav>
+        <main class="main--content">
+             <div class="about--container">
+                <h1>I’m Charles<br>Glad to see<br> you!</h1> 
+             </div> 
+             <div class="description--container">
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+             </div>
+             <p class ="supporting--text">LET’S CONNECT</p>
+             <div class="social--icons">
```

Comment:
[ ] kindly use a `ul` tag to group the icons list, and `li` per icon.🤓

Reply:
@nedjwab well understood I will fix this ASAP.